apps/feelslike/feels_like.star

```diff
@@ -0,0 +1,285 @@
+"""
+Applet: Feels Like
+Summary: A weather display
+Description: An abstract weather display that communicates feeling in a natural way.
+Author: eanplatter
+"""
+
+load("cache.star", "cache")
+load("encoding/json.star", "json")
+load("http.star", "http")
+load("math.star", "math")
+load("random.star", "random")
+load("render.star", "render")
+load("schema.star", "schema")
+
+MAX_COLOR_VALUE = 255
+MAX_ROWS_S = 32
+MAX_ROWS_L = 16
+MAX_COLUMNS_S = 32
+MAX_COLUMNS_L = 16
+DEFAULT_ORIENTATION = "horizontal"
+DEFAULT_SIZE = "s"
+DEFAULT_API = None
+DEFAULT_LAT = "40.730610"  # New York City
+DEFAULT_LON = "-73.935242"  # New York City
+DEFAULT_SPEED = 3
+DEFAULT_LOCATION = """
+{
+	"lat": "40.6781784",
+	"lng": "-73.9441579",
+	"description": "Brooklyn, NY, USA",
+	"locality": "Brooklyn",
+	"place_id": "ChIJCSF8lBZEwokRhngABHRcdoI",
+	"timezone": "America/New_York"
+}
+"""
+
+def main(config):
+    API_KEY = config.get("api_key", DEFAULT_API)
```

Comment:
```suggestion
    random.seed(time.now().unix // 30)
    API_KEY = config.get("api_key", DEFAULT_API)
```

Reply:
This would make the random graphics deterministic over 30s time windows.

Reply:
I have implemented this change.